ddtrace/tracer/log.go

```diff
@@ -55,6 +55,7 @@ type startupInfo struct {
 	PartialFlushEnabled         bool                         `json:"partial_flush_enabled"`          // Whether Partial Flushing is enabled
 	PartialFlushMinSpans        int                          `json:"partial_flush_min_spans"`        // The min number of spans to trigger a partial flush
 	Orchestrion                 orchestrionConfig            `json:"orchestrion"`                    // Orchestrion (auto-instrumentation) configuration.
+	FeatureFlags                map[string]struct{}          `json:"feature_flags"`
```

Comment:
We can probably convert this to `[]string` for the log. It looks a bit off like this.
```
"feature_flags":{"discovery":{}}
``` 
vs 
```
"feature_flags":["discovery"]
```